### DISCUSSION TITLE
UploadToAttribute at the class level

### DISCUSSION THREAD

**Discussion by bgiromini** [ServiceStack/Discuss#63] [Ideas]
I am really enjoying this feature, especially with the auto-populated file metadata that gets stored in the database.

I would suggest however that this feature work at the dto/class level because my intended use case is to upload the file(s) as soon as the user selects them. However my data model is very simple, so unlike the FileRazor example, I don't have the need to create a parent record before storing the file metadata in a child record. All I need to do is upload the file and store the file metadata.

**Comment by mythz** [comment]
I don't understand how this would be supposed to work on a Request DTO, can you please provide a proposed example of what you mean and what the expected behavior should be and where it should save the information.

**Comment by bgiromini** [comment]
After writing this up and re-evaluating how this feature works, I was able to solve my problem.
